cdap-ui/app/cdap/components/PipelineSummary/RunsHistoryGraph/index.js

```diff
@@ -33,7 +33,7 @@ import CopyableRunID from 'components/PipelineSummary/CopyableRunID';
 import SortableStickyTable from 'components/SortableStickyTable';
 import ee from 'event-emitter';
 import EmptyMessageContainer from 'components/PipelineSummary/EmptyMessageContainer';
-import isEqual from 'lodash/isEqual';
+import {isEqual, isNil} from 'lodash';
```

Comment:
This will include the entire lodash library. `lodash/isNil` will include only the isNil part of the library. Would be better to have two imports.